### ISSUE TITLE
Is it possible to be SSR friendly?

### ISSUE THREAD

**Issue by csantos1113** [AndreasFaust/react-raster#10]
When using responsive values for a lot of our props (`colspan` included. For example `colspan={[6, 6, 12, 12]}`) we can see a jump behavior because the currentColspan is calculated in javascript based on the current breakpoint.

I wonder if it's possible to have all of this to live inside `@media` css (similar to all other props), instead of depending on JS to calculate it.

**Note:**
* I tried to fix it in react-raster directly, but it requires quite a lot of time for me understand how all it works, without breaking other things.

### Environment:
```
"next": "10.0.5",
"react": "17.0.1",
"react-dom": "17.0.1",
"react-raster": "7.0.0",
```

![2021-01-27 11 01 59](https://user-images.githubusercontent.com/9648559/106018088-2c9cde80-608f-11eb-94d5-60cca96d712d.gif)

This is how it looks with JS enabled (after JS is loaded)
<img width="1548" alt="Screen Shot 2021-01-27 at 11 03 01 AM" src="https://user-images.githubusercontent.com/9648559/106018244-4f2ef780-608f-11eb-950c-fc8e5908d410.png">

This is how it looks with JS disabled (meaning: the html SSR generated)
<img width="1536" alt="Screen Shot 2021-01-27 at 11 03 13 AM" src="https://user-images.githubusercontent.com/9648559/106018245-4f2ef780-608f-11eb-8c39-1f7020b05f50.png">

**Comment by csantos1113** [comment]
I would try to come back to this issue over the weekend when I'm more free

**Comment by AndreasFaust** [comment]
You are right. I fixed it with the new version `7.0.2`.
Thank you very much and enjoy your weekend!
